src/styles/index.scss

```diff
@@ -1,3 +1,235 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500;700&display=swap';
+
+$primary-color: #00acdc;
+$dark: #060b35;
+$blue: #00acdc;
+$gray: #616070;
+$light-gray: #f3f3f3;
+$white: #fff;
+
+@mixin star {
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+@mixin button-buy {
+  width: 100%;
+  height: 40px;
+  border-radius: 5px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-transform: uppercase;
+  font-weight: 700;
+  text-decoration: none;
+  font-size: 14px;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+}
```

Comment:
The `font-size` on the `body` element is set to `10px`, which is quite small and may be hard to read. Ensure this is intentional and adheres to the design requirements.